### ISSUE TITLE
Wrong signature threshold while performing txs

### ISSUE THREAD

**Issue by alxnode** [anoma/namada#3832]
Binary giving 2 attempts to type correct password
In case 1st attempt was unsuccessful but 2nd was correct it is performing transaction that will fail
Looks like it is expecting to me multisig wallet

Steps to reproduce
Jail any validator
```
./namadac unjail-validator  --validator tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw --node "https://rpc.knowable.run"
Looking-up public key of tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw from the ledger...
Enter your decryption password: **incorrect**
Enter your decryption password: **correct**
Transaction added to mempool.
Transaction hash: 302E8D5C5BB3D60702F2639E03F3EAF9EE0F169B2E3FEE43EF1D5628A2DF1A78
Transaction E0FA279E337A4FD1EEBE10C1C740D132896AA543E6A55B73606A244AAB619E63 was rejected by VPs: [
  "tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw"
]
Errors: [
  [
    "tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw",
    "Invalid transaction section signature: RuntimeError: The section signature is invalid: signature threshold not met: (0 < 1)"
  ]
]
Changed keys: [  "#tnam1qgqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc8j2fp/validator/#tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw/state/last_update",  "#tnam1qgqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc8j2fp/validator/#tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw/state/lazy_map/data/0000000000G0O",  "#tnam1qgqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc8j2fp/validator/#tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw/state/lazy_map/data/0000000000G0Q",  "#tnam1qgqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc8j2fp/validator/#tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw/state/lazy_map/data/0000000000G2O",  "#tnam1qgqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc8j2fp/validator/#tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw/state/oldest_epoch",  "#tnam1qgqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc8j2fp/validator_set_positions/lazy_map/data/0000000000G2O/data/#tnam1qykv7yvcx8eyjld4kuumg0whwjq8usulwy7uhlsw",  "#tnam1qgqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc8j2fp/validator_sets/consensus/lazy_map/data/0000000000G2O/data/00000000000000000000000000000000000000000004PR35PN9G/data/0000000000000"
]
```
This works with any type of txs
```
./namadac transparent-transfer --source tnam1qzxharc68vmgnfam4jpjg5jusme8xr9axq4m5cr7 --target tnam1qp4j3zcxs7qgfg6st7929txmkup8arpryyprnftl --amount 1 --token NAM --memo test_1  --signing-keys tnam1qzxharc68vmgnfam4jpjg5jusme8xr9axq4m5cr7 --node "https://rpc.knowable.run"
Enter your decryption password: **incorrect**
Enter your decryption password: **correct**
Transaction added to mempool.
Transaction hash: 7E413884B16590DCFAFCBA864945225A871528590914562399428BFB293C54FC
Transaction 59173EC91E2978F8BCB5D3B1835E54D8B63C3C54773A3A0F78B320966537D9B9 was rejected by VPs: [
  "tnam1qzxharc68vmgnfam4jpjg5jusme8xr9axq4m5cr7"
]
Errors: [
  [
    "tnam1qzxharc68vmgnfam4jpjg5jusme8xr9axq4m5cr7",
    "Invalid transaction section signature: RuntimeError: The section signature is invalid: signature threshold not met: (0 < 1)"
  ]
]
Changed keys: [  "#tnam1pyqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqej6juv/#tnam1q87wtaqqtlwkw927gaff34hgda36huk0kgry692a/balance/#tnam1qp4j3zcxs7qgfg6st7929txmkup8arpryyprnftl",  "#tnam1pyqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqej6juv/#tnam1q87wtaqqtlwkw927gaff34hgda36huk0kgry692a/balance/#tnam1qzxharc68vmgnfam4jpjg5jusme8xr9axq4m5cr7"
]

```

**Comment by Fraccaman** [comment]
which version of namada are u using?

**Comment by alxnode** [comment]
> which version of namada are u using?

Namada v0.43.0

**Comment by grarco** [comment]
I believe this has been fixed in #3681 which has been merged to main and will be included in the next release

**Comment by grarco** [comment]
Closing this as it is a duplicate of #3525. @alxnode you'll see the fix for this with the next release
